test/modules/cfem_diffusion/c_diffusion_2d_1a_linear.lua

```diff
@@ -35,27 +35,45 @@ w_vol = logvol.RPPLogicalVolume.Create({xmin=-1000.0,xmax=-0.99999, infy=true, i
 n_vol = logvol.RPPLogicalVolume.Create({ymin=0.99999,ymax=1000.0  , infx=true, infz=true})
 s_vol = logvol.RPPLogicalVolume.Create({ymin=-1000.0,ymax=-0.99999, infx=true, infz=true})
 
-e_bndry = 0
-w_bndry = 1
-n_bndry = 2
-s_bndry = 3
+e_bndry = "0"
+w_bndry = "1"
+n_bndry = "2"
+s_bndry = "3"
```

Comment:
These are supposed to be strings, the C++ code requires `std::string`. This worked since the conversion from `1` to `"1"` was done by the lua layer, but it started to fail when I switched to `InputParameters`.